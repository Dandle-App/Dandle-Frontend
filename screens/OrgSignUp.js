import React from "react";
<<<<<<< HEAD
import KeyboardAvoidingWrapper from "../components/organisms/KeyboardAvoidingWrapper";
import { Text, TextInput } from "react-native";
import { StatusBar } from 'expo-status-bar';
import { StyledContainer, InnerContainer, Header1, 
        Header2, Header3, AppLogo, StlyedButton,
        Pad_h_medium, FlexHoriztal, Pad_h_small,
        Header1_2, InputTextContainer, SignUp} from "../styles/styles";


const OrgSignUp = ({navigation}) => {
  return (
    <KeyboardAvoidingWrapper>
      <StyledContainer>
        <StatusBar style="light" />
        <InnerContainer>
          <SignUp>
            <Header1_2>Company Name</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
            <Header1_2>Company Email</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
            <Header1_2>Company Phone Number</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
            <Header1_2>Org Code</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
            <Header1_2>Password</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
            <Header1_2>Cofirm Password</Header1_2>
            <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
          </SignUp>
          <FlexHoriztal>
              <StlyedButton><Text>Capcha</Text></StlyedButton>
              <StlyedButton><Text>Sign Up</Text></StlyedButton>
          </FlexHoriztal>
        </InnerContainer>
      </StyledContainer>
    </KeyboardAvoidingWrapper>
=======
import { Text, TextInput } from "react-native";
import { StatusBar } from 'expo-status-bar';
import { 
        StyledContainer, InnerContainer, Header1, 
        Header2, Header3, AppLogo, StlyedButton,
        Pad_h_medium, 
        FlexHoriztal,
        Pad_h_small,
        Header1_2,
        InputTextContainer} from "../styles/styles";

const OrgSignUp = () => {
  return (
    <StyledContainer>
      <StatusBar style="light" />
      <InnerContainer>
        <Header1_2>Company Name</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Header1_2>Company Email</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Header1_2>Company Phone Number</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Header1_2>Org Code</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Header1_2>Password</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Header1_2>Cofirm Password</Header1_2>
        <TextInput style={{backgroundColor: "white", width:300, height:40}}/>
        <Pad_h_small/>
        <FlexHoriztal>
            <StlyedButton><Text>Capcha</Text></StlyedButton>
            <StlyedButton><Text>Sign Up</Text></StlyedButton>
        </FlexHoriztal>
      </InnerContainer>
    </StyledContainer>
>>>>>>> Created OrgSignUp.js
  );
}

export default OrgSignUp;