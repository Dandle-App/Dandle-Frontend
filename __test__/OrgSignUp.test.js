import React from 'react';
import {renderer} from 'react-test-renderer';
import 'jest-styled-components'
import OrgSignUp from '../screens/OrgSignUp';

const createTestProps = (props) => ({
  navigation: {
    navigate: jest.fn()
  },
  handleSubmit: {
    values: jest.values
  },
  ...props
});

/* 
Jest Snapshots : https://jestjs.io/docs/snapshot-testing
To update a snapshot due to changed implementation: jest --updateSnapshot
*/
// async snapshot
it('renders correctly', () => {
    let tree 
    let props
    beforeEach(() => {
        props = createTestProps({});
    });
    tree = renderer.create(<OrgSignUp {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });