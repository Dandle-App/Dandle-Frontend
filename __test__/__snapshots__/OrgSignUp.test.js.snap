// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrgSignUp renders correctly 1`] = `
<View
  onLayout={[Function]}
>
  <RCTScrollView>
    <View>
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "#fafafa",
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignSelf": "center",
                "backgroundColor": "#fafafa",
                "flexBasis": 0,
                "flexGrow": 2,
                "flexShrink": 1,
                "paddingBottom": 24,
                "paddingLeft": 24,
                "paddingRight": 24,
                "paddingTop": "25%",
                "width": "100%",
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "justifyContent": "flex-start",
                  "width": "90%",
                },
              ]
            }
          >
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Company Name
              </Text>
              <TextInput
                allowFontScaling={true}
                onChangeText={[Function]}
                placeholder="John Doe"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Company Email
              </Text>
              <TextInput
                allowFontScaling={true}
                keyboardType="email-address"
                onChangeText={[Function]}
                placeholder="johndoe@acme.com"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Company Phone No
              </Text>
              <TextInput
                allowFontScaling={true}
                onChangeText={[Function]}
                placeholder="303 309 9200"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Org Code
              </Text>
              <TextInput
                allowFontScaling={true}
                autoCapitalize="characters"
                onChangeText={[Function]}
                placeholder="ACME101"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Password
              </Text>
              <TextInput
                allowFontScaling={true}
                onChangeText={[Function]}
                placeholder="* * * * * * *"
                rejectResponderTermination={true}
                secureTextEntry={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "height": 35,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 1,
                    "top": 35,
                    "width": 35,
                    "zIndex": 200,
                  }
                }
              >
                <Text />
              </View>
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Confirm Password
              </Text>
              <TextInput
                allowFontScaling={true}
                onChangeText={[Function]}
                placeholder="* * * * * * *"
                rejectResponderTermination={true}
                secureTextEntry={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "height": 35,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 1,
                    "top": 35,
                    "width": 35,
                    "zIndex": 200,
                  }
                }
              >
                <Text />
              </View>
            </View>
            <View>
              <View
                style={
                  Array [
                    Object {
                      "left": "2%",
                      "position": "absolute",
                      "top": "60%",
                      "zIndex": 1,
                    },
                  ]
                }
              >
                <Text />
              </View>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#f38484",
                      "fontSize": 16,
                      "paddingBottom": 0,
                      "paddingLeft": 5,
                      "paddingRight": 5,
                      "paddingTop": 5,
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Address
              </Text>
              <TextInput
                allowFontScaling={true}
                onChangeText={[Function]}
                placeholder="123 Main St"
                rejectResponderTermination={true}
                style={
                  Array [
                    Object {
                      "backgroundColor": "#fafafa",
                      "borderBottomWidth": 3,
                      "borderColor": "#d3d3d3",
                      "color": "#212121",
                      "marginBottom": 35,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 5,
                    },
                  ]
                }
                underlineColorAndroid="transparent"
                value=""
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": undefined,
                  },
                ]
              }
            >
              <View>
                <View
                  style={
                    Array [
                      Object {
                        "left": "2%",
                        "position": "absolute",
                        "top": "60%",
                        "zIndex": 1,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#f38484",
                        "fontSize": 16,
                        "paddingBottom": 0,
                        "paddingLeft": 5,
                        "paddingRight": 5,
                        "paddingTop": 5,
                        "textAlign": "left",
                      },
                    ]
                  }
                >
                  City
                </Text>
                <TextInput
                  allowFontScaling={true}
                  onChangeText={[Function]}
                  placeholder="Denver"
                  rejectResponderTermination={true}
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#fafafa",
                        "borderBottomWidth": 3,
                        "borderColor": "#d3d3d3",
                        "color": "#212121",
                        "marginBottom": 35,
                        "paddingBottom": 10,
                        "paddingLeft": 10,
                        "paddingRight": 10,
                        "paddingTop": 5,
                      },
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
              <View
                style={
                  Array [
                    Object {
                      "width": 20,
                    },
                  ]
                }
              />
              <View>
                <View
                  style={
                    Array [
                      Object {
                        "left": "2%",
                        "position": "absolute",
                        "top": "60%",
                        "zIndex": 1,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#f38484",
                        "fontSize": 16,
                        "paddingBottom": 0,
                        "paddingLeft": 5,
                        "paddingRight": 5,
                        "paddingTop": 5,
                        "textAlign": "left",
                      },
                    ]
                  }
                >
                  State
                </Text>
                <TextInput
                  allowFontScaling={true}
                  onChangeText={[Function]}
                  placeholder="CO"
                  rejectResponderTermination={true}
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#fafafa",
                        "borderBottomWidth": 3,
                        "borderColor": "#d3d3d3",
                        "color": "#212121",
                        "marginBottom": 35,
                        "paddingBottom": 10,
                        "paddingLeft": 10,
                        "paddingRight": 10,
                        "paddingTop": 5,
                      },
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
            </View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": undefined,
                  },
                ]
              }
            >
              <View>
                <View
                  style={
                    Array [
                      Object {
                        "left": "2%",
                        "position": "absolute",
                        "top": "60%",
                        "zIndex": 1,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#f38484",
                        "fontSize": 16,
                        "paddingBottom": 0,
                        "paddingLeft": 5,
                        "paddingRight": 5,
                        "paddingTop": 5,
                        "textAlign": "left",
                      },
                    ]
                  }
                >
                  Country
                </Text>
                <TextInput
                  allowFontScaling={true}
                  onChangeText={[Function]}
                  placeholder="United States"
                  rejectResponderTermination={true}
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#fafafa",
                        "borderBottomWidth": 3,
                        "borderColor": "#d3d3d3",
                        "color": "#212121",
                        "marginBottom": 35,
                        "paddingBottom": 10,
                        "paddingLeft": 10,
                        "paddingRight": 10,
                        "paddingTop": 5,
                      },
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
              <View
                style={
                  Array [
                    Object {
                      "width": 20,
                    },
                  ]
                }
              />
              <View>
                <View
                  style={
                    Array [
                      Object {
                        "left": "2%",
                        "position": "absolute",
                        "top": "60%",
                        "zIndex": 1,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#f38484",
                        "fontSize": 16,
                        "paddingBottom": 0,
                        "paddingLeft": 5,
                        "paddingRight": 5,
                        "paddingTop": 5,
                        "textAlign": "left",
                      },
                    ]
                  }
                >
                  Zip Code
                </Text>
                <TextInput
                  allowFontScaling={true}
                  onChangeText={[Function]}
                  placeholder="90210"
                  rejectResponderTermination={true}
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#fafafa",
                        "borderBottomWidth": 3,
                        "borderColor": "#d3d3d3",
                        "color": "#212121",
                        "marginBottom": 35,
                        "paddingBottom": 10,
                        "paddingLeft": 10,
                        "paddingRight": 10,
                        "paddingTop": 5,
                      },
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
            </View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": undefined,
                  },
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#d64547",
                    "borderRadius": 35,
                    "height": 50,
                    "justifyContent": "space-around",
                    "opacity": 1,
                    "width": "100%",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#f5f5f5",
                        "fontSize": 12,
                        "fontWeight": "bold",
                      },
                    ]
                  }
                >
                  sign up
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
